test/inner-loop/base-inner-loop.sh

```diff
@@ -17,7 +17,10 @@ echo -e "\n> Create new odo component"
 odo create $COMP_NAME
 
 echo -e "\n> Create URL with Minikube IP"
-odo url create --host $(minikube ip).nip.io
+odo url create $COMP_NAME --host $(minikube ip).nip.io --ingress
```

Comment:
Why did we pass the `$COMP_NAME` to url create?   I thought our idea was to use the `ep1` url already in the devfile, and that odo had changed the logic to make sure that was used in a case like this.

Reply:
I'm guessing you were trying to keep the next part of the test:
> curl http://$COMP_NAME.$(minikube ip).nip.io/health/live
as similar as possible. 

I think it'd be slightly better to test the 'ep1' endpoint URL, since that is a bit of a convenience shortcut.

Reply:
The issue with the failing tests is to that by issuing an `odo url create --host ...` command without a name, we allowed odo to pick the name. The default url name odo picks appears be different now than what it was originally. Before it was <component>-9080. Now it appears to be just <ep1>. 

So, in order to prevent future failures due to default name changes, i chose to explicitly give the `odo url create...` command the name of <component>.

The URLs that the tests build were originally coded to use the created URL with the default name (<component>-9080).